inference-engine/include/ie_version.hpp

```diff
@@ -74,6 +74,17 @@ struct Version {
             major = _major;
             minor = _minor;
         }
+
+        /**
+         * @brief A copy operator
+         * @param other An object to copy
+         * @return A copy
+         */
+        ApiVersion& operator=(const ApiVersion& other) {
```

Comment:
Replace copy ctor above with `ApiVersion(const ApiVersion & v) = default;` or try to remove copy ctor at all (C++ compiler will create ["Implicitly-declared copy constructor"](https://en.cppreference.com/w/cpp/language/copy_constructor)).

Reply:
Msvc complains about deprecated api in automatically generated code